src/main/java/ch/njol/skript/Skript.java

```diff
@@ -886,14 +899,17 @@ public static boolean isRunningCraftBukkit() {
 	 * @return Whether this server is running Minecraft <tt>major.minor</tt> or higher
 	 */
 	public static boolean isRunningMinecraft(final int major, final int minor) {
+		if (minecraftVersion.compareTo(new Version(666)) == 0) { updateMinecraftVersion(); } // Make sure minecraftVersion is properly assigned
```

Comment:
```suggestion
		if (minecraftVersion.compareTo(new Version(666)) == 0)
			updateMinecraftVersion(); // Make sure minecraftVersion is properly assigned
```
same for other two

Why not use equals here?
You should also make a static final field like UNKNOWN_VERSION containg Version(666)

Reply:
:+1:
As for the using compareTo is to avoid minecraftVersion NPE see https://gist.github.com/AyhamAl-Ali/9117c5b879e6e1657a33dc76b41d4146#file-skript-version-equals-npe